modules/tinymce/CHANGELOG.md

```diff
@@ -29,6 +29,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Split toolbar buttons incorrectly had nested `tabindex="-1"` attributes #TINY-7879
 - Fixed notifications rendering in the wrong place initially and when the page was scrolled #TINY-7894
 - Fixed an exception getting thrown when the number of `col` elements didn't match the number of columns in a table #TINY-7041 #TINY-8011
+- The table selection state could become incorrect after selecting a noneditable table cell #TINY-8053
```

Comment:
"become incorrect" sounds odd but I don't really have a better suggestion

Reply:
I can't think of a better way to phrase it so am just going to leave it for now.